sidebars.json

```diff
@@ -183,7 +183,8 @@
       "contributing/glossary",
       "contributing/debugging",
       "contributing/troubleshooting",
-      "contributing/making-a-pull-request"
+      "contributing/client-asset-tunnelling",
+      "contributing/adding-your-first-test"
```

Comment:
I'm not clear what's happening with making-a-pull-request and adding-your-first-test

Reply:
This was in error - I meant to simply add this new item. Fixed, thanks.